Common/Data/Custom/TradingEconomics/TradingEconomicsCalendar.cs

```diff
@@ -242,6 +335,37 @@ public override string ToString()
             return Invariant($"{symbol} ({Country} - {Category}): {Event} : Importance.{Importance}");
         }
 
+        /// <summary>
+        /// Convert this instance to a CSV file (tab delimited)
+        /// </summary>
+        /// <returns>string as CSV</returns>
+        public string ToCsv()
```

Comment:
The docstring still says tab delimited.

BTW, is there a reason why you add some entries with quotes?

Reply:
Yes, we need the quotes in order to escape values that contain the `,` delimiter as part of their values.